crates/backend/src/lib.rs

```diff
@@ -19,42 +21,65 @@ use render::Renderer;
 
 pub async fn run(config: Config) -> anyhow::Result<()> {
     let (sender, mut receiver) = unbounded_channel();
-    let server = Server::init(sender).await?;
+
+    let server = Server::init(sender.clone()).await?;
     info!("Backend: Server initialized");
 
     let (server_internal_channel, mut renderer) = Renderer::init(config)?;
     info!("Backend: Renderer initialized");
 
     let backend_thread = thread::spawn(move || renderer.run());
 
+    let mut scheduler = Scheduler::new();
+    info!("Backend: Scheduler initialized");
```

Comment:
Move this info log into `Sheduler::new()` method

Reply:
i dont think this is a good idea

Reply:
Alright, I'll leave it.